enhancements/operator-hub-check.md

```diff
@@ -0,0 +1,103 @@
+---
+title: operatorhub-bundle-validation
+authors:
+  - "@camilamacedo86"
+reviewers:
+  - TBD
+approvers:
+  - TBD
+creation-date: 2021-02-19
+last-updated: 2021-02-19
+status: implementable
+---
+
+# Add checks to OperatorHubValidator
+
+This proposal describes a new checks to be implemented in the [operator-framework/api][oper-api] into the [OperatorHubValidator][operatorhubvalidator].
+
+## Release Signoff Checklist
+
+- [x] Enhancement is `implementable`
+- [x] Design details are appropriately documented from clear requirements
+- [x] Test plan is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+
+## Summary
+
+`OperatorHubValidator` checks if the operator is respecting the required criteria to be published in the [OperatorHub.io](https://operatorhub.io/). This proposal describes criteria to be added to it.
+
+Many tools and projects use `OperatorHubValidator` already. Note that the new checks described by this proposal will be available via SDK:
+
+```sh
+To validate a bundle with a superset of requirements for operatorhub.io specifically:
+   
```

Comment:
```sh

Reply:
wow GitHub did something odd with that, I'm just suggesting we should move the start of the sh quoting down to not include the text explanation.

Reply:
The text explanation is part of the tool output because of this is in ```sh

Reply:
I tried to clarify it. Tks

Reply:
@camilamacedo86  I would completely remove the `--help` and description since it doesn't really add much to the explanation. Just show the proposed commands:

```
operator-sdk bundle validate ./bundle --select-optional name=operatorhub
```